src/neo4j_genai/retrievers/text2cypher.py

```diff
@@ -67,15 +81,10 @@ def __init__(
                 if validated_data.neo4j_schema_model
                 else get_schema(validated_data.driver_model.driver)
             )
-        except Neo4jError as e:
-            logger.error("Failed to get schema for Text2CypherRetriever: %s", e.message)
-            raise RuntimeError(
-                f"Failed to get schema for Text2CypherRetriever: {e.message}"
+        except (Neo4jError, DriverError) as e:
+            raise SchemaFetchError(
+                f"Failed to fetch schema for Text2CypherRetriever: {e}"
```

Comment:
`mypy` tells me `e.message` doesn't exist

Reply:
I think it's only defined for `Neo4jError` and not for `DriverError`. If I remember well, that was the reason for splitting the error handling in two.

Reply:
https://github.com/neo4j/neo4j-genai-python/pull/55/commits/b46b32db96c901cbd79be0a91071782d9566d488 I do a check here. I think this is less verbose